### PR TITLE
Wrap Flint's tstbit

### LINKED ISSUE #715: Add test_bit
This would test whether a given bit on an integer is 1 and would complement the already existing clrbit!, setbit! and combit! functions.

### PR DESCRIPTION
**Pull request by wbhart**
Fixes #715

### CODE REVIEW

File: src/flint/fmpz.jl [hunk 1]
```diff
@@ -1988,6 +1989,18 @@ function combit!(x::fmpz, c::Int)
     ccall((:fmpz_combit, libflint), Nothing, (Ref{fmpz}, Int), x, c)
 end
 
+@doc Markdown.doc"""
+    tstbit(x::fmpz, c::Int)
+
+Test bit $i$ of x (numbered from 0) and return 1 or 0 depending if it is set.
```
Comment (rfourquet) [inline src/flint/fmpz.jl:1995]: Maybe return a `Bool`? As `Bool <: Integer`, the result can still be used as an integer, but also as a condition e.g. in an `if` statement.

**Comment by wbhart** [comment]
I'm just following the flint and GMP naming convention here. This is a pretty low level function which isn't going to get much use, so I figure the name isn't too important.

File: src/flint/fmpz.jl [hunk 2]
```diff
@@ -1988,6 +1989,18 @@ function combit!(x::fmpz, c::Int)
...
+"""
+function tstbit(x::fmpz, c::Int)
+   if c < 0
+      return 0
```
Comment (tthsqe12) [inline src/flint/fmpz.jl:1999]:
```suggestion
      return false
```
Also maybe some test code for this with @ inferred ?

File: src/flint/fmpz.jl (hunk 1 above)
Comment (tthsqe12) [inline src/flint/fmpz.jl:1995]:
```suggestion
Return the $i$^th bit of x (numbered from 0) as `true` for 1 or `false` for 0.
```

**Review by tthsqe12 — CHANGES_REQUESTED** [review]

**Comment by thofma** [comment]
Sorry, this needs a rebase.

**Pull request closed without merging**